CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   ([#1690](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1690))
 - Add metrics instrumentation for sqlalchemy
   ([#1645](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1645))
+- Make requests span attributes available to Django sampler
+  ([#1645](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1645))
```

Comment:
There is no `Django` sampler as such in this repo. One may use it for whatever reasons. 
```suggestion
- Make Django request span attributes available for `start_span`. 
  ([#1645](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1645))
```